examples/react/ssr/webpack.config.js

```diff
@@ -1,64 +1,62 @@
 const path = require('path');
 
+const webpack = require('webpack');
```

Comment:
would be nice some other time if we find what the new plug-and-play pure server side rendering solution is? Vite server side rendering plugin maybe?

Reply:
Yes I agree `vite` could work well for this use case.

Reply:
Or just get rid of this example tbh, we shouldn't have to maintain something like this when there are metaframeworks like Next and Remix